src/components/SearchPage/SearchInputComponent.js

```diff
@@ -0,0 +1,84 @@
+import React, { useState } from 'react';
+import InputAdornment from '@material-ui/core/InputAdornment';
+import TextField from '@material-ui/core/TextField';
+import SearchIcon from '@material-ui/icons/Search';
+import { IconButton, makeStyles } from '@material-ui/core';
+import { useDispatch } from 'react-redux';
+import { useDebouncedCallback } from 'use-debounce/lib';
+import { searchQueryChangeAction } from './SearchPageAction';
+import { useEffect } from 'react';
+import { colors } from 'colors';
+import ClearIcon from '@material-ui/icons/Clear';
+
+const useStyles = makeStyles(() => ({
+  multilineColor: {
```

Comment:
What do you mean by `multiline` here?
I think this style should be named `input` and it's applied to the input.

Reply:
Just copy paste from docs.  renamed.